Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 
 [dependencies]
 tracy-client = { version = "0.17.3", default-features = false } # for tracy v0.11.1
+nu-protocol = "*"
```

Comment:
It's better to use a labeled version and bump manually to avoid sudden CI breakages.